src/main/java/mate/academy/spring/mapper/impl/response/ShoppingCartResponseMapper.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartResponseMapper implements
+        DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart> {
+    @Override
+    public ShoppingCartResponseDto toDto(ShoppingCart shoppingCart) {
+        ShoppingCartResponseDto shoppingCartResponseDto = new ShoppingCartResponseDto();
+        shoppingCartResponseDto.setId(shoppingCart.getId());
+        List<Long> ticketIdList = shoppingCart
+                .getTickets()
+                .stream()
+                .map(t -> t.getId())

```

Comment:
You can simply use method ref here for just  `.map(Ticket::getId)` , but it's not so important

Reply:
do as suggested